cairo_programs/cairo_0/bitwise_builtin_test.cairo

```diff
@@ -4,17 +4,11 @@
 from starkware.cairo.common.bitwise import bitwise_and, bitwise_xor, bitwise_or, bitwise_operations
 from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
 
-func main{bitwise_ptr: BitwiseBuiltin*}() {
-    let (and_a) = bitwise_and(12, 10);  // Binary (1100, 1010).
-    assert and_a = 8;  // Binary 1000.
-    let (xor_a) = bitwise_xor(12, 10);
-    assert xor_a = 6;
-    let (or_a) = bitwise_or(12, 10);
-    assert or_a = 14;
+func main{bitwise_ptr: BitwiseBuiltin*}(x: felt, y: felt) {
```

Comment:
afaik the `main` function cannot take parameters in Cairo Zero (doable in Cairo though)

Overall, avoid parameters in test functions

Reply:
Got it, thank  man!